server/test_sudoku.py

```diff
@@ -0,0 +1,42 @@
+from server.sudoku import (
+    read_from_file,
+    validate_list_enries,
+    validate_region,
+    validate_sudoku,
+)
+
+
+def test_validate_list_entries() -> None:
+    test_list = [0 for i in range(81)]
+    assert validate_list_enries(test_list)
```

Comment:
```suggestion
    assert validate_list_entries(test_list)
```

Samma med de övriga i denna funktion. :)